Actions/ScheduleActions/ScheduleActionCreators.js

```diff
@@ -1,11 +1,52 @@
-import { FILTER_PERFORMANCES } from './ScheduleActionTypes';
+import BASE_URL from '../../baseURL';
```

Comment:
Prefer use absolute imports in cases when you need go back. especially few times